recipes/prgenv-gnu/24.11/gh200/config.yaml

```diff
@@ -0,0 +1,7 @@
+name: prgenv-gnu
+spack:
+  commit: releases/v0.23
```

Comment:
This doesn't exist yet... do an rc with `develop` until then?

Reply:
let's do an release candidate for internal testing.

Reply:
`releases/v0.23` now exists, but the `v0.23.0` tag doesn't exist yet.